documentation.md

```diff
@@ -178,10 +178,25 @@ Mozilla.Cookies.removeItem('mozilla-traffic-cop-original-referrer');
 
 Traffic Cop requires three JavaScript files:
 
-1. `mozilla-cookie-helper.js` (included in the `src` directory)
-2. `mozilla-traffic-cop.js`
+1. `@mozmeao/cookie-helper` (install via [NPM](https://www.npmjs.com/package/@mozmeao/cookie-helper))
+2. `@mozmeao/trafficcop` (install via [NPM](https://www.npmjs.com/package/@mozmeao/trafficcop))
```

Comment:
**Question:** When is best to tell users to install the cookie helper on their own via NPM vs having CookieHelper as a dependency of TrafficCop?

Reply:
Shamelessly I think for now we need to keep the dependencies as installed separately, mostly for bedrock. As the cookie helper lib is used for many different things there. It needs to exist in an entirely separate bundle to traffic cop.

I’m not sure if anyone outside our own team using Traffic Cop. Perhaps if we see that change, we could figure something better out.